rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -664,3 +711,274 @@ def test_applying_all_upgrade(globaldb: GlobalDBHandler):
+            up_to_version=None,
+            conflicts={
+                Asset(conflict['identifier']): 'remote'
+                for conflict in conflicts
+            },
+        ) is None
+
+    with (
+        globaldb.conn.read_ctx() as old_db_cursor,
+        globaldb.packaged_db_conn().read_ctx() as packaged_db_cursor,
+    ):
+        ignored_identifiers = {
+            identifiers[0]
+            for cursor in (old_db_cursor, packaged_db_cursor)
+            for identifiers in cursor.execute(
+                'SELECT identifier FROM evm_tokens WHERE '
+                f'protocol IN ({",".join(["?"] * len(IGNORED_PROTOCOLS))}) OR '
+                'address in (SELECT value FROM general_cache)',
+                list(IGNORED_PROTOCOLS),
+            ).fetchall()
+        }
+
+        (
+            (updated_assets, updated_underlying_db_assets, updated_collections, updated_multiasset_mappings),  # noqa: E501
+            (packaged_assets, packaged_underlying_db_assets, packaged_collections, packaged_multiasset_mappings),  # noqa: E501
+        ) = (
+            (
+                {
+                    identifier: DbToken(
+                        address=address,
+                        chain=chain,
+                        token_kind=token_kind,
+                        decimals=decimals,
+                        name=name,
+                        symbol=symbol,
+                        started=started,
+                        swapped_for=swapped_for,
+                        coingecko=coingecko,
+                        cryptocompare=cryptocompare,
+                        protocol=protocol,
+                    )
+                    for (identifier, address, chain, token_kind, decimals, name, symbol, started, swapped_for, coingecko, cryptocompare, protocol)  # noqa: E501
+                    in cursor.execute(
+                        'SELECT A.identifier, B.address, B.chain, B.token_kind, B.decimals, '
+                        'C.name, A.symbol, A.started, A.swapped_for, A.coingecko, '
+                        'A.cryptocompare, B.protocol FROM evm_tokens AS B JOIN '
+                        'common_asset_details AS A ON B.identifier = A.identifier '
+                        'JOIN assets AS C on C.identifier=A.identifier',
+                    )
+                    if identifier not in ignored_identifiers
+                }, cursor.execute(
+                    'SELECT identifier, weight, parent_token_entry FROM underlying_tokens_list',
+                ).fetchall(), {
+                    collection_id: (name, symbol)
+                    for (collection_id, name, symbol)
+                    in cursor.execute('SELECT id, name, symbol FROM asset_collections')
+                }, cursor.execute(
+                    'SELECT collection_id, asset FROM multiasset_mappings',
+                ).fetchall(),
+            )
+            for cursor in (old_db_cursor, packaged_db_cursor)
+        )
+
+    assets_to_update = []
```

Comment:
is this assets or collections?

Reply:
It's for all three.. assets, collections, and multiasset_mappings